From 4399a557f408c3c47634e6bec1922e0399ea39ab Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Mon, 30 Oct 2023 19:25:27 -0400
Subject: [PATCH 42/54] Add a prop to use linear backlight control

---
 .../settings/brightness/BrightnessController.java    | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java b/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
index 649f8db89bc0..8f57677b5da7 100644
--- a/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
+++ b/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
@@ -369,9 +369,15 @@ public class BrightnessController implements ToggleSlider.Listener, MirroredBrig
                 : MetricsEvent.ACTION_BRIGHTNESS;
         minBacklight = mBrightnessMin;
         maxBacklight = mBrightnessMax;
-        final float valFloat = MathUtils.min(
+        float valFloat = MathUtils.min(
                 convertGammaToLinearFloat(value, minBacklight, maxBacklight),
                 maxBacklight);
+        if (android.os.SystemProperties.getBoolean("persist.sys.phh.linear_brightness", false)) {
+            android.util.Log.e("PHH", "Linear brightness val " + value + " from " + minBacklight + " to " + maxBacklight + " makes " + valFloat);
+            valFloat = value / 65536.0f;
+        }
+        final float finalValFloat = valFloat;
+
         if (stopTracking) {
             // TODO(brightnessfloat): change to use float value instead.
             MetricsLogger.action(mContext, metric,
@@ -385,8 +391,8 @@ public class BrightnessController implements ToggleSlider.Listener, MirroredBrig
         if (!tracking) {
             AsyncTask.execute(new Runnable() {
                     public void run() {
-                        logBrightnessChange(mDisplayId, valFloat, false);
-                        mDisplayManager.setBrightness(mDisplayId, valFloat);
+                        logBrightnessChange(mDisplayId, finalValFloat, false);
+                        mDisplayManager.setBrightness(mDisplayId, finalValFloat);
                     }
                 });
         }
-- 
2.25.1

