From 63a39ed1c87597f423dd5f6594fd1bb2ec6981df Mon Sep 17 00:00:00 2001
From: kindle4jerry <kindle4jerry@gmail.com>
Date: Sun, 6 Feb 2022 14:42:18 +0800
Subject: [PATCH 3/8] treble: sepolicy: fix honor v9 no voice

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 sepolicy/file_contexts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index faa7256..4b08746 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -19,3 +19,5 @@
 /efs u:object_r:efs_file:s0
 
 /dev/smcinvoke u:object_r:smcinvoke_device:s0
+
+/dev/nxp_smartpa_dev    u:object_r:media_rw_data_file:s0
-- 
2.32.0 (Apple Git-132)

