From fc0ef706d89e5d7cb46e71407610570e8f26e510 Mon Sep 17 00:00:00 2001
From: xiaoleGun <1592501605@qq.com>
Date: Sat, 26 Feb 2022 12:01:01 +0800
Subject: [PATCH 9/9] treble: proper target names

---
 generate.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/generate.sh b/generate.sh
index 352fe12..07e65b5 100644
--- a/generate.sh
+++ b/generate.sh
@@ -65,6 +65,9 @@ for part in a ab;do
 				fi
 
 				target="treble_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
+				if [ "$1" == "miku" ];then
+					target="miku_${target}"
+				fi
 
 				baseArch="$arch"
 				if [ "$arch" = "a64" ];then
-- 
2.32.0 (Apple Git-132)

