From 2112c099672a129e04e886e272e4992a1c8841b8 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Mon, 29 Nov 2021 17:49:13 -0500
Subject: [PATCH 3/6] Don't unmount rw-system.sh binds

Change-Id: If9132c21defa8b09879b79a70794c5275d6852d0
---
 VolumeManager.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/VolumeManager.cpp b/VolumeManager.cpp
index a7c87a3a..10952897 100644
--- a/VolumeManager.cpp
+++ b/VolumeManager.cpp
@@ -1032,6 +1032,7 @@ int VolumeManager::unmountAll() {
     while ((mentry = getmntent(fp)) != NULL) {
         auto test = std::string(mentry->mnt_dir);
         if ((StartsWith(test, "/mnt/") &&
+             !StartsWith(test, "/mnt/phh") &&
 #ifdef __ANDROID_DEBUGGABLE__
              !StartsWith(test, "/mnt/scratch") &&
 #endif
-- 
2.25.1

