From 10aa398c75926556e8f1c735705c44bdac87315f Mon Sep 17 00:00:00 2001
From: xiaoleGun <1592501605@qq.com>
Date: Sun, 6 Feb 2022 14:35:16 +0800
Subject: [PATCH 2/9] treble: sepolicy: Remove fsck SELinux labels

---
 sepolicy/file_contexts | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 999ff97..11dd447 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -7,9 +7,6 @@
 /system/bin/asus-motor u:object_r:phhsu_exec:s0
 /system/bin/xiaomi-touch u:object_r:phhsu_exec:s0
 
-#/system/bin/fsck\.exfat                 u:object_r:fsck_exec:s0
-/system/bin/fsck\.ntfs                  u:object_r:fsck_exec:s0
-
 /bt_firmware(/.*)?      u:object_r:bt_firmware_file:s0
 
 /sec_storage(/.*)?      u:object_r:teecd_data_file:s0
-- 
2.32.0 (Apple Git-132)

