From bed44a027f78a4a48a8143ebcd72fcd38e7f1d1f Mon Sep 17 00:00:00 2001
From: kindle4jerry <kindle4jerry@gmail.com>
Date: Sun, 6 Feb 2022 14:42:18 +0800
Subject: [PATCH 03/15] treble: sepolicy: fix honor v9 no voice

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 sepolicy/file_contexts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index beb639a..a0bfa2d 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -1,3 +1,5 @@
+/dev/nxp_smartpa_dev    u:object_r:media_rw_data_file:s0
+
 /system/bin/phh-su                   u:object_r:phhsu_exec:s0
 /system/bin/vndk-detect			u:object_r:vndk_detect_exec:s0
 /system/etc/usb_audio_policy_configuration.xml	u:object_r:vendor_configs_file:s0
-- 
2.32.1 (Apple Git-133)

