From ba59e9c496f8803acbd9752320e0b8380b236ab5 Mon Sep 17 00:00:00 2001
From: kindle4jerry <kindle4jerry@gmail.com>
Date: Sun, 6 Feb 2022 14:46:06 +0800
Subject: [PATCH 4/9] treble: fix huawei phone unkonw fingerprint

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 rw-system.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index 5e275af..f28677b 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -789,7 +789,7 @@ if [ -c /dev/dsm ];then
 
     mkdir -p /data/sec_storage_data
     chown system:system /data/sec_storage_data
-    chcon -R u:object_r:teecd_data_file:s0 /data/sec_storage_data
+    chcon -R u:object_r:teecd_data_file_system:s0 /data/sec_storage_data
 
     if mount | grep -q " on /sec_storage " ; then
         # /sec_storage is already mounted by the vendor, don't try to create and mount it
@@ -797,12 +797,12 @@ if [ -c /dev/dsm ];then
         # the fingerprint daemon (running as system) cannot access it.
         chown -R system:system /sec_storage
         chmod -R 0660 /sec_storage
-        chcon -R u:object_r:teecd_data_file:s0 /sec_storage
+        chcon -R u:object_r:teecd_data_file_system:s0 /sec_storage
     else
         # No /sec_storage provided by vendor, mount /data/sec_storage_data to it
         mount /data/sec_storage_data /sec_storage
         chown system:system /sec_storage
-        chcon u:object_r:teecd_data_file:s0 /sec_storage
+        chcon u:object_r:teecd_data_file_system:s0 /sec_storage
     fi
 fi
 
-- 
2.32.0 (Apple Git-132)

