From 1cf9d6e8d64e14b56b46073461fb72e04ca472f1 Mon Sep 17 00:00:00 2001
From: AndyCGYan <GeForce8800Ultra@gmail.com>
Date: Mon, 7 Feb 2022 14:49:31 +0800
Subject: [PATCH 05/15] treble: Don't handle apns-conf

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 base-pre.mk | 3 ---
 base.mk     | 4 ----
 2 files changed, 7 deletions(-)

diff --git a/base-pre.mk b/base-pre.mk
index 6a317e4..e69de29 100644
--- a/base-pre.mk
+++ b/base-pre.mk
@@ -1,3 +0,0 @@
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
diff --git a/base.mk b/base.mk
index 423fe1d..6b4dba3 100644
--- a/base.mk
+++ b/base.mk
@@ -17,10 +17,6 @@ PRODUCT_COPY_FILES += \
     device/phh/treble/bluetooth/audio/config/sysbta_audio_policy_configuration.xml:$(TARGET_COPY_OUT_SYSTEM)/etc/sysbta_audio_policy_configuration.xml \
     device/phh/treble/bluetooth/audio/config/sysbta_audio_policy_configuration_7_0.xml:$(TARGET_COPY_OUTY_SYSTEM)/etc/sysbta_audio_policy_configuration_7_0.xml
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-
 SYSTEM_EXT_PRIVATE_SEPOLICY_DIRS += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += \
     device/phh/treble/overlay \
-- 
2.32.1 (Apple Git-133)

