From 669cbcb5150e45ae11a82ec09e89c1ca01c295f4 Mon Sep 17 00:00:00 2001
From: Alberto Ponces <ponces26@gmail.com>
Date: Sun, 16 Oct 2022 21:02:52 +0100
Subject: [PATCH 15/15] Remove force disable of A2DP offload property

---
 vndk.rc | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/vndk.rc b/vndk.rc
index d1fffde..515cfb4 100644
--- a/vndk.rc
+++ b/vndk.rc
@@ -52,15 +52,6 @@ on property:persist.sys.phh.disable_soundvolume_effect=*
 on property:ro.sf.lcd_density=*
     restart surfaceflinger
 
-on property:persist.sys.phh.disable_a2dp_offload=0
-    setprop persist.sys.phh.disable_a2dp_offload false
-
-on property:persist.sys.phh.disable_a2dp_offload=1
-    setprop persist.sys.phh.disable_a2dp_offload true
-
-on property:persist.sys.phh.disable_a2dp_offload=*
-    setprop persist.bluetooth.bluetooth_audio_hal.disabled ${persist.sys.phh.disable_a2dp_offload}
-
 on property:init.svc.ril-proxy=stopped && property:persist.sys.phh.restart_ril=true
     start ril-proxy
 
-- 
2.32.1 (Apple Git-133)

