From 32f8666f9d1f26e3dd98bf4b9332d01495dc6e72 Mon Sep 17 00:00:00 2001
From: xiaoleGun <1592501605@qq.com>
Date: Sat, 28 Jan 2023 10:39:48 +0800
Subject: [PATCH 15/18] treble: Use new gapps

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 gapps.mk | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/gapps.mk b/gapps.mk
index c61752b..e5d0126 100644
--- a/gapps.mk
+++ b/gapps.mk
@@ -52,9 +52,8 @@ endif
 #endif
 
 #Miku UI Gapps
-ifneq ($(wildcard vendor/gapps),)
-$(call inherit-product, vendor/gapps/common/common-vendor.mk)
+ifneq ($(wildcard vendor/gms),)
+$(call inherit-product, vendor/gms/gms.mk)
 PRODUCT_PACKAGES += \
-       phh-overrides \
-
+    phh-overrides
 endif
-- 
2.32.1 (Apple Git-133)

