From 6ecfc54a73a81f20384a257745cacdf9317ea9fb Mon Sep 17 00:00:00 2001
From: xiaoleGun <1592501605@qq.com>
Date: Sat, 28 Jan 2023 11:38:30 +0800
Subject: [PATCH 17/18] treble: Not have kernel

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 miku.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/miku.mk b/miku.mk
index 492268c..b934a6f 100644
--- a/miku.mk
+++ b/miku.mk
@@ -2,6 +2,9 @@
 $(call inherit-product, vendor/miku/build/product/miku_product.mk)
 -include vendor/miku/build/core/config.mk
 
+# Kernel
+TARGET_NO_KERNEL := true
+
 # Maintaier
 MIKU_MASTER := xiaoleGun
 
-- 
2.32.1 (Apple Git-133)

