From 35a578f547950de3c8398ae5a9b3856a0c21d34b Mon Sep 17 00:00:00 2001
From: AndyCGYan <GeForce8800Ultra@gmail.com>
Date: Sun, 6 Feb 2022 14:58:02 +0800
Subject: [PATCH 2/3] build_soong: Disable generated_kernel_headers

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 build/soong/Android.bp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index d056b6c..30eb125 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -20,9 +20,6 @@ bootstrap_go_package {
 miku_generator {
     name: "generated_kernel_includes",
 
-    // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
-
     // Directories that can be imported by a cc_* module generated_headers property
     export_include_dirs: [
         "usr/audio/include/uapi",
@@ -30,10 +27,6 @@ miku_generator {
         "usr/include/audio/include/uapi",
         "usr/techpack/audio/include"
     ],
-
-    // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [ "Makefile", "include/**/*", "arch/$(KERNEL_ARCH)/include/**/*", "techpack/audio/include/**/*"],
 }
 
 cc_library_headers {
-- 
2.32.0 (Apple Git-132)

